Renderer.cs

```diff
@@ -32,14 +39,103 @@ public SpriteBatchSettings(SpriteSortMode ssm, BlendState bs, SamplerState ss, D
     public readonly Effect Effect;
 }
 
+public abstract class IFont
```

Comment:
The naming convention `ISomething` is reserved to interfaces and should not used by abstract classes.

This could be named just `Font`, or if that conflicts with something in the dependencies, then `FontBase` or `AbstractFont` are good candidates.

Alternatively, since this class currently only declares an API and has zero concrete implementation, as-is this class could just be changed to an actual interface.

Reply:
I honestly didn't even notice it's not an interface, it seems weird to me that this is not an interface when it's used like one. Considering that -- should just use an interface directly.